assets/css/admin.css

```diff
@@ -1,3 +1,11 @@
orter-error-log-table td:last-child,.woocommerce-progress-form-wrapper .woocommerce-exporter .wc-importer-error-log-table th:last-child,.woocommerce-progress-form-wrapper .woocommerce-exporter .wc-importer-mapping-table td:last-child,.woocommerce-progress-form-wrapper .woocommerce-exporter .wc-importer-mapping-table th:last-child,.woocommerce-progress-form-wrapper .woocommerce-importer .wc-importer-error-log-table td:last-child,.woocommerce-progress-form-wrapper .woocommerce-importer .wc-importer-error-log-table th:last-child,.woocommerce-progress-form-wrapper .woocommerce-importer .wc-importer-mapping-table td:last-child,.woocommerce-progress-form-wrapper .woocommerce-importer .wc-importer-mapping-table th:last-child{padding-right:24px}.woocommerce-exporter-wrapper .wc-progress-form-content .wc-importer-error-log-table .wc-importer-mapping-table-name,.woocommerce-exporter-wrapper .wc-progress-form-content .wc-importer-mapping-table .wc-importer-mapping-table-name,.woocommerce-exporter-wrapper .woocommerce-exporter .wc-importer-error-log-table .wc-importer-mapping-table-name,.woocommerce-exporter-wrapper .woocommerce-exporter .wc-importer-mapping-table .wc-importer-mapping-table-name,.woocommerce-exporter-wrapper .woocommerce-importer .wc-importer-error-log-table .wc-importer-mapping-table-name,.woocommerce-exporter-wrapper .woocommerce-importer .wc-importer-mapping-table .wc-importer-mapping-table-name,.woocommerce-importer-wrapper .wc-progress-form-content .wc-importer-error-log-table .wc-importer-mapping-table-name,.woocommerce-importer-wrapper .wc-progress-form-content .wc-importer-mapping-table .wc-importer-mapping-table-name,.woocommerce-importer-wrapper .woocommerce-exporter .wc-importer-error-log-table .wc-importer-mapping-table-name,.woocommerce-importer-wrapper .woocommerce-exporter .wc-importer-mapping-table .wc-importer-mapping-table-name,.woocommerce-importer-wrapper .woocommerce-importer .wc-importer-error-log-table .wc-importer-mapping-table-name,.woocommerce-importer-wrapper .woocommerce-importer .wc-importer-mapping-table .wc-importer-mapping-table-name,.woocommerce-progress-form-wrapper .wc-progress-form-content .wc-importer-error-log-table .wc-importer-mapping-table-name,.woocommerce-progress-form-wrapper .wc-progress-form-content .wc-importer-mapping-table .wc-importer-mapping-table-name,.woocommerce-progress-form-wrapper .woocommerce-exporter .wc-importer-error-log-table .wc-importer-mapping-table-name,.woocommerce-progress-form-wrapper .woocommerce-exporter .wc-importer-mapping-table .wc-importer-mapping-table-name,.woocommerce-progress-form-wrapper .woocommerce-importer .wc-importer-error-log-table .wc-importer-mapping-table-name,.woocommerce-progress-form-wrapper .woocommerce-importer .wc-importer-mapping-table .wc-importer-mapping-table-name{width:50%}.woocommerce-exporter-wrapper .wc-progress-form-content .wc-importer-error-log-table .wc-importer-mapping-table-name .description,.woocommerce-exporter-wrapper .wc-progress-
```

Comment:
This needs to be changed in the SCSS source files, and the admin.css file is then built from those.

For now, leaving the changes here is fine until someone can help who has this build process set up.

Reply:
Yes, I thought it was something like that. I think @bahiirwa was doing that before and he's offered to take a look when he has time.